NitroxModel/Core/NitroxEnvironment.cs

```diff
@@ -87,7 +87,7 @@ public static bool IsReleaseMode
     /// <summary>
     ///     Gets the command line arguments as passed to the program on start.
     /// </summary>
-    public static string[] CommandLineArgs => commandLineArgs ??= Environment.GetCommandLineArgs().Skip(1).ToArray();
+    public static string[] CommandLineArgs => commandLineArgs ??= Environment.GetCommandLineArgs().Skip(1).Select(p => p.Trim('\'')).ToArray();
```

Comment:
What is that about ?

Reply:
Windows handles command args differently than Linux. Windows removes the ' character around an arg. 
For Linux, this needs to be done manually to match behavior.

(will change the code with comment to reflect)